docs/lifecycle.md

```diff
@@ -72,9 +72,9 @@ Incoming GraphQL Websocket subscription data
                      errors ◀─┴─▶ Subscription Execution
                                     │
                                     │
-                                    └─▶ preGatewaySubscriptionExecution Hook(s)
+                                    └─▶ preGatewaySubscriptionExecution Hook(s) (using the module @mercuriusjs/gateway)
                                             │
-                                   errors ◀─┴─▶ Gateway Subscription Execution(s)
+                                   errors ◀─┴─▶ Gateway Subscription Execution(s) (using the module @mercuriusjs/gateway)
```

Comment:
These should be moved to the gateway readme.

Reply:
I've removed all the hooks that are from the gateway and simplified the lifecycle. It's documented here https://github.com/mercurius-js/mercurius-gateway#Lifecycle